package.json

```
@@ -17,11 +17,22 @@
   },
   "scripts": {
     "start": "eleventy --serve --quiet",
+    "build": "eleventy",
+    "pack": "gulp",
```

```suggestion
    "build": "eleventy && gulp",
```

@nlopin думаешь лучше так?

---

@pepelsbey мне кажется, что да — лучше так. Но есть другой вопрос, я опишу его в отдельном треде чуть ниже.

---

да, так нормес 👍